### DISCUSSION TITLE
Weird API issue 2023-06-07

### DISCUSSION THREAD

**Discussion by youzer-name** [teslamate-org/teslamate#3234] [General]
Just posting this to report a strange issue with the Tesla API in case it happens again.

I have Teslamate setup both locally and on Google Cloud Platform. (I set up the cloud version recently to test it, but noticed that the local version was picking up status changes from the cars that the GCP version missed... so I'm sticking with local for now and just using the free GCP instance as a backup)

I use Uptime Kuma to monitor the status of both sites. That tool accesses each site periodically and looks for the name of one of my cars in the response. If the site doesn't respond or the response doesn't contain the car name, it marks the site as down.

On 2023-06-07 around 12:00 UTC, both sites started reporting as down. When I checked them manually to see what the issue was, both sites were up, but the names of the cars had disappeared from the Teslamate home page. Only the model names were displayed. I checked my Tesla app and the car names were still showing there.

I had planned to manually pull data from the API via a command line to see what the response looked like, but before I got around to doing that, on 2023-06-08 at about 23:15 UTC both sites started reporting as back up. I checked and they were again showing the car names.

I suppose this could have just been an issue with my account, but it seems more likely something changed in the API response that either dropped the car names or reported them in a way that Teslamate didn't parse them. I'm curious if anyone else noticed anything.

**Comment by cwanja** [comment]
You are likely seeing the same behavior as reported in #3183 

> I suppose this could have just been an issue with my account, but it seems more likely something changed in the API response that either dropped the car names or reported them in a way that Teslamate didn't parse them. I'm curious if anyone else noticed anything.

Several are, as linked above, including myself. But as of this morning (approximately 12AM CST), cars name returned and has not dropped.

**Comment by STSUKX** [comment]
Same Thing here. Started this morning. Car still unamed at the moment via API.

**Comment by haexpo** [comment]
Same problem here:

Log shows this:

teslamate_1     | 2023-06-11 11:45:27.618 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/*removed*/vehicle_data -> 408 (9104.728 ms)
teslamate_1     | 2023-06-11 11:45:27.619 [warning] TeslaApi.Error / %{"error" => "{\"error\": \"timeout\"}", "error_description" => "", "response" => nil}
teslamate_1     | 2023-06-11 11:45:27.619 car_id=2 [error] Error / :unknown
teslamate_1     | 2023-06-11 11:45:55.218 car_id=2 [info] Suspending logging

**Comment by tobiasehlert** [comment]
This issue is resolved with commit https://github.com/adriankumpf/teslamate/commit/6af97bafc8cfee6bd22911a1014dae8dcbe1f419, so you'll have to wait for a new release to get it resolved (or switch to `edge`).
